### ISSUE TITLE
Broken awt image

### ISSUE THREAD

**Issue by klpx** [sksamuel/scrimage#125]
Hi! Thank you for your library!
I caugth some issue between awt converting and ImageIO writer.
I am trying to save scrimage using ImageIO (after converting it back to awt) 
Here is my original picture:
![tire](https://cloud.githubusercontent.com/assets/2909427/21855656/3c6a3ba0-d830-11e6-9478-91c665575306.jpg)

And my code:
```
import javax.imageio.ImageIO
import java.io.File
import com.sksamuel.scrimage.Image
object TestApp extends App {
  val dir = s"/home/hsslbch/Documents/example"
  def in: BufferedImage = ImageIO.read(new File(s"$dir/tire.jpg"))

  {
    val outDirect = new File(s"$dir/tire_direct.jpg")
    ImageIO.write(in, "jpg", outDirect)
  }
  {
    val outDirect = new File(s"$dir/tire_direct.png")
    ImageIO.write(in, "png", outDirect)
  }
  {
    val outConvert = new File(s"$dir/tire_convert.jpg")
    val converted = Image.awtToScrimage(in).awt
    ImageIO.write(converted, "jpg", outConvert)
  }
  {
    val outConvert = new File(s"$dir/tire_convert.png")
    val converted = Image.awtToScrimage(in).awt
    ImageIO.write(converted, "png", outConvert)
  }
}
```

Everything is ok when i just save awt Image just given from ImageIO.read. Also everything is ok when i convert scrimage to awt and save it to png.
But not jpg (tire_convert.jpg), that is what i got
![tire_convert](https://cloud.githubusercontent.com/assets/2909427/21856031/84c39b5c-d831-11e6-84c0-0839acf03948.jpg)

This picture is filled with black in GNOME Image Viewer. And it is red colored in Google Chrome (Linux).

**Comment by klpx** [comment]
Just duplicate screenshot to show what i see:
![image](https://cloud.githubusercontent.com/assets/2909427/21856129/e6cabb8c-d831-11e6-9b3e-d63f2a9d6dfc.png)

**Comment by sksamuel** [comment]
I see Red as well (linux). I'll take a look thanks.

**Comment by poslegm** [comment]
I think that source of problem is data type. By default `fromAwt` method uses `BufferedImage.TYPE_INT_ARGB`. But this image has `BufferedImage.TYPE_3BYTE_BGR` type. I wrote this type explicitly and image was converted correct.

**Comment by mirosval** [comment]
So after some investigation it turns out that its `ImageIO`'s fault, it will happily write 4 channel AWT image into `jpg` but will make a CMYK Jpeg, you can download this pink car tire image and see for yourself (On macOS open in Preview and then cmd+i)

**Comment by sksamuel** [comment]
This is fixed in 4.0
